traitlets/utils/importstring.py

```diff
@@ -23,7 +49,7 @@ def import_item(name):
     mod : module object
        The module that was imported.
     """
-    
+    name = _to_module_str(name)
```

Comment:
I think we've already got the function we want as `py3compat.cast_bytes_py2`, which converts unicode to str on Python 2 and leaves unicode/str alone on Python 3. I don't think we want to be converting bytes to str on Python 3.